scripts/install-dev.sh

```diff
@@ -15,6 +15,20 @@ printf "=====================Waiting for Openshift Gitops start up and running..
 kubectl wait --for=condition=Ready pods --all -n openshift-gitops
 echo "=====================Openshift Gitops deploy successful!"
 
+printf "=====================Patch Openshift Gitops Subscription to enable default instance which will prevent the custom instance we created to be deleted by operator ...\n"
```

Comment:
I saw we use the `echo` and `printf` func to print the log. maybe we can choose one to do it.